ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -7,15 +7,41 @@
         <exclude pattern="/apps/acs-commons/components/utilities/page/clientlibs/css/generated"/>
         <exclude pattern="/apps/acs-commons/components/utilities/oak-index-manager/clientlibs/css/generated"/>
         <exclude pattern="/apps/acs-commons/components/utilities/bulk-workflow-manager/clientlibs/css/generated"/>
-        <include pattern="/apps/acs-commons/components/utilities/page-compare(.*)?"/>
         <exclude pattern="/apps/acs-commons/components/utilities/page-compare/clientlibs/css/generated"/>
-        <include pattern="/apps/acs-commons/components/utilities/version-compare(.*)?"/>
         <exclude pattern="/apps/acs-commons/components/utilities/version-compare/clientlibs/css/generated"/>
+        <exclude pattern="/apps/acs-commons/config(\..*)?(/.*)?" />
```

Comment:
@kwin i missed this one originally - so OSGi configs are now moved into a content folder in the `content` (`all`) package? Is this idiomatic? I thought typically these should be in a `ui.config` package using the "regular" packaging methods?

Reply:
This is the recommended layout: https://jackrabbit.apache.org/filevault/packagetypes.html

Reply:
Gotcha - but alternatively we could also put them in a `ui.config` page that is a `container` type - you just added them to this project/package since this is the existing `container` type project we have (vs. making a new one).

Reply:
That would be possible as well, but requires a new Maven module.